app/views/layouts/_content.html.haml

```diff
@@ -16,10 +16,9 @@
   = render :partial => "layouts/vertical_navbar"
 
   .container-fluid.container-pf-nav-pf-vertical.container-pf-nav-pf-vertical-with-tertiary.resizable-sidebar.max-height{:style => "overflow: hidden !important"}
-    .row.toolbar-pf#toolbar
+    .row.toolbar-pf#toolbar.miq-toolbar-menu
       .col-md-12
-        .toolbar-pf-actions
-          = render_toolbars(calculate_toolbars)
```

Comment:
Thought it as well, didn't quite know if it's safe to remove. But I'll remove it.